src/gui/history/qgshistoryprovider.cpp

```diff
@@ -0,0 +1,19 @@
+/***************************************************************************
+                            qgshistoryprovider.cpp
+                            -------------------------
+    begin                : April 2019
```

Comment:
Love a 2019 idea coming to fruition :)

Reply:
Heh... the actual QEP predates even that file!!